EventFilter/CTPPSRawToDigi/src/CTPPSPixelDataFormatter.cc

```diff
@@ -148,6 +148,12 @@ void CTPPSPixelDataFormatter::interpretRawData(  bool& errorsInEvent, int fedId,
     int dcol = (ww >> m_DCOL_shift) & m_DCOL_mask;
     int pxid = (ww >> m_PXID_shift) & m_PXID_mask;
 
+    if(dcol<0||dcol>=26||pxid<2||pxid>161){
```

Comment:
are there constants already defined for these ranges 0-25 and 2-160?
If so, please use them instead of the magic numbers.